pkg/cmd/output.go

```diff
@@ -192,17 +192,11 @@ func makeAllocationTable(allocationType string, allocations map[string]kubecost.
 	for _, alloc := range allocations {
 
 		// This variable exists to scale costs by the active window
-		// if the query is not marked historical
-		var histScaleFactor float64
+		var histScaleFactor float64 = 1
 
-		if isHistorical {
+		if !projectToMonthlyRate {
```

Comment:
I think this should be `if projectToMonthlyRate` for this to make sense, though it means flipping the truth value for all of your invocations of this function in `namespace.go`, `deployment.go`, etc.